targets/TARGET_RENESAS/TARGET_VK_RZ_A1H/gpio_irq_api.c

```diff
@@ -71,6 +71,7 @@ static const PinMap PinMap_IRQ[] = {
 #else
 static const PinMap PinMap_IRQ[] = {
     {P9_1,  IRQ0, 4},
+    {P7_8,  IRQ1, 8},
```

Comment:
It is forgotten external Interrupt pin, which was missed at the creation of the SDK for that platform.